src/talk2159012.lua

```diff
@@ -0,0 +1,17 @@
+speakerSchiller = self.as_speaker(2159008)
+self.say("The experiment is going well, quite well. The endless supply of Rue is certainly speeding things along. Joining the Black Wings was a wise decision, a wise decision indeed. Muahaha!")
+speakerSchiller.say("I say, you have great foresight about these things.")
+self.say("The android the Black Wings wanted will be completed soon. Oh yes, very soon. Then, the next stage will begin! I will conduct an experiment wilder than their wildest dreams!")
+speakerSchiller.say("Pardon? The next stage?")
+self.say("Teeheehee, do you still not comprehend what I''m trying to create? Look around! Here''s a clue: it''s eons more interesting than a simple android. Eons more interesting.")
```

Comment:
Double apostrophe found at "I''m'. This happens in some other resistance tutorial files as well.